### PR TITLE
FEATURE: FET-478 Add FOSSA configuration

### PR DESCRIPTION
**Pull request by vlastikcz**
---

**Proposed release type:** major|minor|patch (see points 6. - 8. of the [semver spec](https://semver.org/#semantic-versioning-specification-semver))

# PR checklist

- [ ] Change was tested using dev-release in Analytical Designer and Dashboards (if applicable).
- [ ] Change is described in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Migration guide (for major update) is written to [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] The proposed release type is appropriate (see the comment in [PR template](../blob/master/.github/pull_request_template.md))

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-oss-check/129/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/570/ : SUCCESS in 1m 33s

**Comment by vlastikcz** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-oss-check/130/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/571/ : SUCCESS in 1m 42s

File: gdc_fossa.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
+scan_image: "harbor.intgdc.com/tool/gdc-fossa-cli-frontend-node:node-12.16.1-yarn-1.22.4"
```
Comment (pbenes) [inline gdc_fossa.yaml:2]: Just curious if the node/yarn version should be in sync with the used image for building. If yes, have you considered some possibility (may not even be feasible) to link image version from https://github.com/gooddata/gooddata-js/blob/master/docker/.config#L1 or from some common ancestor space?

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-oss-check/132/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/573/ : SUCCESS in 1m 24s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-fossa-scanning-zuul-el7/6/ : FAILURE in 9s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-js-merge-compliance-check/132/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/574/ : SUCCESS in 1m 37s

**Pull request merged**